### ISSUE TITLE
There is no errors detected but project fails to build 

### ISSUE THREAD

**Issue by Edna-White** [FlutterFlow/flutterflow-issues#15]
Issue tracker is **ONLY** used for reporting bugs. New feature suggestions and questions should be discussed on Community or submitted through our user feedback form.

Your issue may already be reported! Please search in the [issue tracker](../) before creating one.

Please **thumbs up** this issue if you have also experienced it. You may also add more information if there is something relevant that was not mentioned. However, please refrain from comments that are not constructive, like "I have this problem too", etc.

## Expected behavior (required)

Without errors detected, the test build should complete(succeed)

## Current behavior (required)

Without issues found(presented), the test build keeps failing.

## To Reproduce (required)

1. Confirm that no errors are found on the top bar in project (https://app.flutterflow.io/project/selfiew-sfwyfg)
2. Run test build by clicking test button
3. Loading... (preparing testing session for 20-30 s) 
4. Build Failed screen appears 

## Context (required)

I cannot build test so go on to complete my project any more. 

## Screenshots / recordings

https://app.flutterflow.io/debug/QzX344aJHFDT8ozJYNcN
<img width="1071" alt="Screen Shot 2022-09-26 at 10 11 57 PM" src="https://user-images.githubusercontent.com/114411423/192285423-a8cc3d36-d644-4313-a534-246190f9891c.png">

## Your environment

* Version of FlutterFlow used: v1
* Platform (e.g. Web, MacOS Desktop): MacOS Desktop
* Browser name and version:  Chrome 105.0.5195.125 (Official Build) (x86_64)
* Operating system and version (desktop or mobile): MacOS Monterey 12.5.1

**Comment by Edna-White** [comment]
Hi @bulgariamitko 
Thanks for the update.

I couldn’t understand correctly what I should do for you to reproduce the issue.

Should I create a new project duplicated from the project(Selfiew)? Or just create a new? 

If you mean the former, I just created one named ‘for issue reproduce’.
But it seems this duplicated project has no firebase project yet so has errors on it. 
Is it okay? Or should I create another firebase project and itergrate with this?

**Comment by Edna-White** [comment]
@bulgariamitko I just created new firebase project and completed setup and integration with flutterflow project (for test). 
Maybe you can test this for the issue reproduction. 
https://app.flutterflow.io/project/for-issue-reproduce-edvchm (Public)

**Comment by katherineqian** [comment]
Hi @Edna-White, thanks for using our new issue tracker! This space is intended to be for reporting **specific** bugs when you know exactly what is not working. Since your project has a build failure for unknown reasons, it doesn't qualify as a specific bug. This issue is better suited for contacting support, where they can help you figure out what's going on (chat or email support [at] flutterflow.io). I will add more guidance to the issue tracker for this and again, appreciate your submission.

**Comment by Edna-White** [comment]
Hi @katherineqian, 
I'm very sad about this situation and the unresolved issue which has been unresolved for more than 1 week. 
Sin 9/21 I had already reported this issue by contacting support via messenger and email several times. But I couldn't get any feeback or support. No matter how many times I tried, I couldn't find an answer, so I came here.

If it doesn't be clarified here also, what should I do?
Very time-consuming and I cannot go on my project anymore. Very disappointing situation.
